### DISCUSSION TITLE
Documentation clarification: Do you need to uninstall snapd everywhere

### DISCUSSION THREAD

**Discussion by Rillian-Grant** [ltsp/ltsp#215] [Q&A]
The docs say this:
```
In case you end up choosing Ubuntu MATE 18.04, @alkisg recommends running the following commands after installation, to avoid some problematic packages:

apt purge --yes --auto-remove indicator-application mate-hud snapd
apt install --yes synaptic
```
Snapd is installed on every ubuntu flavour. Is this specific to Ubuntu MATE?

**Comment by alkisg** [comment]
LTSP doesn't have any problems at all with snap. It's just my personal preference for the best Ubuntu experience. Feel free to completely ignore it.

**Comment by Rillian-Grant** [comment]
Wouldn't it be better to point this out and/or separate it from the things that you absolutely have to do to get it to work on Ubuntu MATE

**Comment by Rillian-Grant** [comment]
If you describe the intention behind that paragraph of documentation I would be happy to rewrite it.

**Comment by alkisg** [comment]
Indicator-application is causing duplicate network-manager applets, and complete hangs on some cases where fcitx is also installed.
Mate-hud was very unstable, crashing etc. and wastes a lot of RAM.
Snap too wastes a lot of RAM and confuses some users that can't detect where the packages come from, normal Ubuntu or snap repositories. E.g.they install snaps per user and expect to find it in the other user accounts on the clients.
Once snap is uninstalled, boutique is no longer available, so synaptic is needed to install software. On the plus side, mate-welcome goes away too, which is a bit annoying on multiuser systems.

Considering that many users are still using LTSP with old hardware, those are my general suggestions that we're using in Greek schools and saved us from a lot of repetitive issues in forums and IRC.
On the other hand, many users are happily using all the above applications, so I do not want to badmouth them.

Thanks, I updated my phrasing in the installation page, hopefully it's a bit better now.

**Comment by Rillian-Grant** [comment]
The update looks good.

I just rewrote that section to try to make it more explicit about why you would want to remove each package: ltsp/ltsp.org#4

**Comment by alkisg** [comment]
Thank you Rillian-Grant, I commented there in the pull request.

**Comment by Rillian-Grant** [comment]
Oh well,  feel free to use my work for anything else.
